sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,193 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    r"""Coulomb kinetic friction with Stribeck and viscous effects.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = (\mu_k f_n + (\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}) \text{sign}(v) + \sigma  v
+
+    where :math:`\mu_k` is the coefficient of kinetic friction, :math:`\mu_s` is the
+    coefficient of static friction, :math:`f_n` is the normal force, :math:`v` is the
+    relative velocity, :math:`v_s` is the Stribeck friction coefficient, and
+    :math:`\sigma` is the viscous friction constant.
+
+    The default friction force is :math:`F = \mu_k f_n`.
+    When specified, the actuator includes:
+
+    - Stribeck effect: :math:`(\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}`
+    - Viscous effect: :math:`\sigma v`
+
+    Notes
+    ======
+    The actuator makes the following assumptions:
+    - The actuator assumes slip.
+    - The normal force is assumed to be a non-negative scalar.
+    - The resultant friction force is opposite to the velocity direction.
+    This actuator has been tested for straightforward motions, like a block sliding
+    on a surface.
```

Comment:
The itemize is not properly formatted, as can be seen [here](https://output.circle-artifacts.com/output/job/301721e8-d8d1-4f1c-9f0c-ae3ed1596d10/artifacts/0/doc/_build/html/modules/physics/mechanics/api/actuator.html#sympy.physics.mechanics.actuator.CoulombKineticFriction). You forgot to add (an) empty line(s).